app/views/shared/check_answers/_vehicles.html.erb

```diff
@@ -40,22 +36,17 @@
       <%= summary_list.with_row(html_attributes: { id: "app-check-your-answers__vehicle__#{index}_change_link" }) do |row| %>
         <%= row.with_key(text: t(".providers.vehicle_index", sequence: index + 1), classes: "govuk-!-width-one-half govuk-heading-m") %>
         <%= row.with_value(text: nil) %>
-            <%= row.with_action(
-                  text: t("generic.change"),
-                  href: providers_legal_aid_application_means_vehicle_detail_path(@legal_aid_application, vehicle),
-                  visually_hidden_text: "vehicle #{index + 1}",
-                  ) %>
         <%= row.with_action(
               text: t("generic.change"),
               href: providers_legal_aid_application_means_vehicle_detail_path(@legal_aid_application, vehicle),
-              visually_hidden_text: t(".providers.vehicle_index", sequence: index + 1).downcase,
+              visually_hidden_text: "vehicle #{index + 1}",
```

Comment:
looks like the rebase remove the use of the translation, or was it intentional?

Reply:
D'oh! 🤦‍♂️